### ISSUE TITLE
When training (eval) predicted image has nothing in common with GT

### ISSUE THREAD

**Issue by ExtReMLapin** [haoyGONG/LP-Diff#6]
Hello,
Title, here are the images from the eval folder : 

<img width="224" height="112" alt="Image" src="https://github.com/user-attachments/assets/cd4a99ff-1e68-4d2d-9e80-add179be645f" />
<img width="224" height="112" alt="Image" src="https://github.com/user-attachments/assets/ceab8932-20bf-467d-a46e-e99f399ae201" />
<img width="224" height="112" alt="Image" src="https://github.com/user-attachments/assets/4cc2160b-d77f-485d-a727-6065c83a8917" />
<img width="224" height="112" alt="Image" src="https://github.com/user-attachments/assets/63a942d0-9539-433a-bb8c-9455dd4d41b0" />
<img width="224" height="112" alt="Image" src="https://github.com/user-attachments/assets/bd92c2e2-b7a1-4e16-9b7b-6e0bf8b5d859" />

**Comment by ExtReMLapin** [comment]
I'm pretty sure this repository is a scam

**Comment by ExtReMLapin** [comment]
Fixed it in this commit, MTA module was initialized from scratch and not having loss calculated so there was no way it could work

https://github.com/haoyGONG/LP-Diff/commit/02e42a04d5bdc2a96357480f809fe9b9bfc800a5

**Comment by chisphung** [comment]
Hi @ExtReMLapin , im running this experiment for a report, i just want to take the honest result from this repo. Could i ask you that the result you commented in the first turn was the initial code or modified MTA?

**Comment by chisphung** [comment]
Oh, i realized loss did not change after over 300 epochs 🤡

**Comment by ExtReMLapin** [comment]
First image was official code with only the augmentations fixed

**Comment by ExtReMLapin** [comment]
This repository is literally a SCAM

**Comment by ExtReMLapin** [comment]
@chisphung to get it to work you need to apply my MTA fix and augmentations fix.

Then only train the MTA module alone, than freeze it and use it in the whole model training.

But even with that, you won't get good results, it's mostly when you feed it 3 images literally going lazy (because that's how ai works) and copy the best image quality

**Comment by chisphung** [comment]
@ExtReMLapin  Thank you for your clarification, becaues i just want to reproduce and write to my report, so that is okay. But I realized that their setup in the paper was different from the released code, and the main problem was the learning rate.

**Comment by ExtReMLapin** [comment]
if you even manage to get it working, feel free to ping me !

**Comment by chisphung** [comment]
Ok, thank you bro
